lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+  def initialize
+  end
+ 
+  def create_vehicles(registration_location)
+    registration_location.map do |registration|
+      Vehicle.new(
```

Comment:
You are pulling in the correct data with `registration_location` but when we create a new `Vehicle` object what datatype should the argument be?
```ruby
Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
```
I looked at the `vehicle_spec` to get this. How could you update 
```ruby
Vehicle.new(...)
```
on line 7?